src/resources/js/Pages/Welcome.vue

```diff
@@ -3,30 +3,26 @@
 
         <div id="page-wrapper" class="h-screen w-full flex flex-wrap">
             <div class="bg-blue w-full text-center text-white p-1">
-                &nbsp;
+               Important Covid-19 Information. <a href="https://coronavirus.ohio.gov/wps/portal/gov/covid-19/home">Learn More.</a>
             </div>
             <div class="border-b border-blue-500 w-full p-1">
                 <h1 class="logo float-left text-center text-2xl mx-auto my-1 ml-2">Vaccinate OH</h1>
 
                 <div class="float-right px-6 py-2 z-40">
+                    <inertia-link href="/" class="text-blue font-bold ml-4">Home</inertia-link>
 
-                    <inertia-link href="/dashboard" class="text-sm text-gray-700 underline ml-4">Stats</inertia-link>
+                    <inertia-link href="https://info.vaccinateoh.org" class="text-blue font-bold ml-4">About Us</inertia-link>
```

Comment:
I'm going to change these from inertia-link to regular old a tags. Inertia loads links without a page reload, which isn't really necessary. I know you were just following what was here already.